students/Sean_Tasaki/Lesson2/GridPart3.py

```diff
@@ -0,0 +1,15 @@
+def gridPatternPart3(rc, s):
```

Comment:
It's a good idea for readability to use meaningful names for function parameters